.github/workflows/build.yaml

```diff
@@ -1,20 +1,21 @@
 name: "build"
 on: [push, pull_request]
+
 jobs:
   build:
     name: build
     runs-on: ubuntu-latest
     steps:
       - name: Setup BATS
-        uses: mig4/setup-bats@v1
-        with:
-          bats-version: 1.7.0
```

Comment:
What will the bats version it will be? In my Ubuntu 22.04, it's `1.2.1-3build1`, which seems pretty old.

The original setup-bats gets the bats from https://github.com/bats-core/bats-core/archive/v${version}.tar.gz

Reply:
Hi @holyspectral ,
Thanks for the review, Let me find a way to retain the current version. As far as I know, our original setting doesn’t have a specific reason for this particular version.

Reply:
Thanks for the advice, please take a look